### ISSUE TITLE
how to reset learning rate if total epoch set to a small number

### ISSUE THREAD

**Issue by ILG2021** [SWivid/F5-TTS#1029]
### Checks

- [x] This template is only for research question, not usage problems, feature requests or bug reports.
- [x] I have thoroughly reviewed the project documentation and read the related paper(s).
- [x] I have searched for existing issues, including closed ones, no similar questions.
- [x] I am using English to submit this issue to facilitate community communication.

### Question details

I use gradio to train. The learning rate save to model. But I found the epochs set is too small. I have change it, but learning rate can not to change. any way to solve this problem. want finetune cli can set total steps instead of epoch. it is hard to count how many epochs need.

**Comment by SWivid** [comment]
- Remove scheduler related stuff from checkpoint file

or 

- Command out temporarily scheduler related state_dict loading in trainer.py

**Comment by ILG2021** [comment]
> * Remove scheduler related stuff from checkpoint file
> 
> or
> 
> * Command out temporarily scheduler related state_dict loading in trainer.py

I have change the weight file to do it.
